test/test_mixeddofhandler.jl

```diff
@@ -643,6 +643,26 @@ function test_vtk_export()
     rm(filename*".vtu") # clean up
 end
 
+function test_celliterator_on_true_subdomain_smoketest()
+    grid = generate_grid(Hexahedron, (3,3,3))
```

Comment:
```suggestion
    grid = generate_grid(Hexahedron, (2,2,2))
```

Reply:
I am not seeing the issue here, could you explain?

Reply:
Just a general idea that I think we should minimize the workload during tests since they already take long. Of course, this is **very** minor, but a lot of minor things add up...